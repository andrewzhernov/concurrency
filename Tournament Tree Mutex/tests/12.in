5 20000 10
