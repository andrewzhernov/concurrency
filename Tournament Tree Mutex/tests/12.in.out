Testing with parameters: n_threads=5, n_iters=20000, n_shareds=10
Starting 5 threads ... 
Done
Joining 5 threads ... 
Done
Checking test invariant ... 
Done
OK
Testing with parameters: n_threads=5, n_iters=20000, n_shareds=10
Starting 5 threads ... 
Done
Joining 5 threads ... 
Done
Checking test invariant ... 
Done
OK
Testing with parameters: n_threads=5, n_iters=20000, n_shareds=10
Starting 5 threads ... 
Done
Joining 5 threads ... 
Done
Checking test invariant ... 
Done
OK
